scripts/zkai_competitor_landscape_commitment_check.py

```
@@ -0,0 +1,100 @@
+
+
+def load_payload(path: pathlib.Path) -> dict[str, Any]:
+    if path.is_symlink():
+        raise CompetitorLandscapeCommitmentError(f"evidence path must not be a symlink: {path}")
+    if not path.is_file():
+        raise CompetitorLandscapeCommitmentError(f"evidence path is not a file: {path}")
+    try:
+        with path.open("r", encoding="utf-8") as handle:
+            payload = json.load(handle)
+    except json.JSONDecodeError as err:
+        raise CompetitorLandscapeCommitmentError(f"invalid JSON in {path}: {err}") from err
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)